.github/workflows/build.yml

```diff
@@ -72,6 +72,7 @@ jobs:
             target/release/keygen
             target/release/permissionless-builder
             target/release/nasty-client
+            target/release/test-node
```

Comment:
I think since we eventually want to distribute this we should call the binary something that identifies the binary more easily out of the context of this repo. I don't have a great idea but something like `espresso-test-node` is probably better than just `test-node`.

Reply:
renamed to `espresso-dev-node`